test/integration/project_test.go

```diff
@@ -106,12 +106,14 @@ func TestProjectCreate_ApplicationDoesntExist_FailAndReturnErrorMessage(t *testi
 
 func TestProjectCreate_ApplicationExists_CreateProjectSuccessfully(t *testing.T) {
 
-	err, _ := executeCommand(
+	err, outBuffer := executeCommand(
 		t, "project", "create", flag(params.FormatFlag),
 		printer.FormatJSON, flag(params.ProjectName), projectNameRandom,
 		flag(params.ApplicationName), "my-application",
 	)
-
+	createdProject := wrappers.ProjectResponseModel{}
```

Comment:
there is in assertion on the project creation there?

Reply:
there is an error and it checked in line117

Reply:
we should have an assertion and not just error... can you add please? (if it's not complex)

Reply:
done